momento-sdk/src/main/java/momento/sdk/IAuthClient.java

```diff
@@ -0,0 +1,16 @@
+package momento.sdk;
+
+import java.io.Closeable;
+import java.util.concurrent.CompletableFuture;
+import momento.sdk.auth.accessControl.DisposableTokenScope;
+import momento.sdk.auth.accessControl.ExpiresIn;
+import momento.sdk.responses.auth.GenerateDisposableTokenResponse;
+
+public interface IAuthClient extends Closeable {
```

Comment:
@nand4011 she originally had this extending `AutoCloseable`, but changed it to match your existing stuff. I'm actually wondering if we shouldn't change them all to `AutoCloseable` though. Thoughts?

Reply:
I don't have a problem with that.